client/components/CreateProfile.js

```diff
@@ -0,0 +1,238 @@
+      uploadedImageURI: 'x',
+      name: '',
+      base64: ''
+    }
+  
+  static navigationOptions = {
+    title: 'Create profile',
+  };
+
+  //   fetch(`${BASE_URL}/${this.props.me}`, {
+  //     method: "POST",
+  //     headers: { 'Content-Type': 'application/json' }
+  //   })
+  //   .then(rawData => rawData.json())
+  //   .then( (parsedContacts) => {
+  //     this.props.storeContacts(parsedContacts);
+  //   });
+  // };
+
+  postImage = async (base64Payload, name) => {
+    let payload = JSON.stringify({
+      name: name,
+      imageBase64: base64Payload
+     });
+
+     console.log('postImage');
+
+    await fetch(`${BASE_URL}/image/save`, {
+      method: "POST",
+      headers: { 'Content-Type': 'application/json' },
+      body: payload
+    })
+    // .then(rawData => rawData.json())
+    .then(rawData => rawData.text())
+    .then( (imageName) =>  {
+      const uploadedImageURI = `${BASE_URL}/${imageName}`;
+
+      this.setState({uploadedImageURI: uploadedImageURI});
+    })
+    .catch( (err) => console.log('Image POST error :', err));
+
+  };
+
+  _pickImage = async () => {
+    let result = await ImagePicker.launchImageLibraryAsync({
+      allowsEditing: true,
+      aspect: [4, 3],
+      base64: true
+    });
+
+
+    // console.log(result);
+    
+    if (!result.cancelled) {
+      // this.setState({ image: result.uri });
+      this.setState({ image: result.uri, base64: result.base64 });
+      // console.log(converted);
+    }
+  };
+  
+  render() {
+    const { image } = this.state;
+    return (
+      <View style={styles.container}>
+
+        <Formik
+          initialValues={{ fName: '', lName: '', email: '', occupation: '', date: '' }}
+
+          onSubmit={ async ({fName, lName, email, occupation, date}) => {
+            
+            Keyboard.dismiss();
+            await this.postImage(this.state.base64, fName);
+            const { uploadedImageURI } = this.state;
+
+            let newProfile = createNewProfile(uploadedImageURI, fName, lName, email, occupation, date);
+
+            this.props.navigation.navigate('CreateProfileAddInfo', {
+              newProfile: newProfile
+            });
+          }
+          }>
+          {({ handleChange, handleSubmit, values }) => (
+
+
+            <ScrollView 
+              contentContainerStyle={styles.wrapper}
+              showsVerticalScrollIndicator={false}
+              >
+
+              <Text style={{ fontSize: 12, marginBottom: 30 }}>Photo</Text>
+              <View style={{ flex: 1, justifyContent: 'center' }}>
+                {image &&
+                  <Avatar
+                    onPress={this._pickImage}
+                    source={{ uri: image }}
+                    large
+                    avatarStyle={{ width: 110, height: 110, marginBottom: 20 }}
+                    rounded
+                  />}
+              </View>
+
+              <TextField
```

Comment:
with all these text fields I feel like you could have done a map over state so that way there isn't a lot of repetitive code.
{this.state.objThatHoldsAllReqInfo.map((el) => {
  <TextField
  //values like label, placeholder, values etc
});